app/src/main/res/values/strings.xml

```diff
@@ -61,5 +61,5 @@
     <string name="addressBookHelpText">If this option is enabled, reserving with the tablet requires entering a valid name that is synced to the accounts address book contacts.</string>
     <string name="googleAccountHelp">Account or accounts used for searching rooms.</string>
     <string name="defaultTitleForReservation">Reserved</string>
-    <string name="hintText">Hint: You can use your tablet\'s address book contacts for reserving.</string>
+    <string name="hintText">Enter name or contact</string>
```

Comment:
It has to do with the rearrangement of the reservation pop-up. I removed
the separate text field for the hint (instructions). In it's place, I'm
using the built-in widget's hint text, which shows the hint prompt inline
on the edit field. But to do so, it has to be brief, so I shorten the whole
text. I think it's a better approach, but we can always put it back if you
prefer.

On Tue, Sep 8, 2015 at 2:18 PM, Ben Klang <notifications@github.com> wrote:

> In app/src/main/res/values/strings.xml
> <https://github.com/bklang/meeting-room-tablet/pull/12#discussion_r38981637>
> :
>
> > @@ -61,5 +61,5 @@
> >      <string name="addressBookHelpText">If this option is enabled, reserving with the tablet requires entering a valid name that is synced to the accounts address book contacts.</string>
> >      <string name="googleAccountHelp">Account or accounts used for searching rooms.</string>
> >      <string name="defaultTitleForReservation">Reserved</string>
> > -    <string name="hintText">Hint: You can use your tablet\'s address book contacts for reserving.</string>
> > +    <string name="hintText">Enter name or contact</string>
>
> Is there a reason you took out the note about the address book?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bklang/meeting-room-tablet/pull/12/files#r38981637>.
>